src/concrete/ml/onnx/convert.py

```diff
@@ -167,6 +168,10 @@ def get_equivalent_numpy_forward_from_onnx(
             forward.
         check_model (bool): set to True to run the onnx checker on the model.
             Defaults to True.
+        lsbs_to_remove (List[int]): Contains the values of the least significant bits to
```

Comment:
same, typing should be `Optional[List[int]`

Reply:
same reason, mypy